.github/PULL_REQUEST_TEMPLATE/BugFix.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug Fix
+about: Add a bug fix this project!
+title: "[Bug Fix] Short Form Title (50 chars or less.)"
+labels: bug
+assignees: 'WithoutPants, bnkai, Leopere'
+
+---
+<!-- Please make sure to read https://github.com/stashapp/stash/docs/CONTRIBUTING.md and check that you understand and have followed it as best as possible -->
+# Pre-Submission Checklist
+* [ ] I have read [CONTRIBUTING.md](/docs/CONTRIBUTING.md)
```

Comment:
Requiring the user to tick this box is unnecessary, and as mentioned elsewhere, adds tasks to the pull request entry in the pull request page. The comment is sufficient.

Reply:
Fixing in a new commit

Reply:
Should be removed properly.